### DISCUSSION TITLE
【1868140】潜水员戴夫

### DISCUSSION THREAD

**Discussion by 746650396** [pjy612/SteamManifestCache#149] [Share]
appid:1868140
```
1868141;dec1482ee88e2a62d7fddedf0e75c6abdef55d2884440de6508bef83044198d8
```
[1868141_3133638072090399802.zip](https://github.com/pjy612/SteamManifestCache/files/13761304/1868141_3133638072090399802.zip)
电子邮箱：2811036643@qq.com
第一次分享，看看对不对？

**Comment by 746650396** [comment]
code：AEA2417A40DA1A8B

**Reply by pjy612** [reply]
密钥呢？

**Reply by 746650396** [reply]
密钥在压缩包里面。这个是吗：dec1482ee88e2a62d7fddedf0e75c6abdef55d2884440de6508bef83044198d8

**Reply by pjy612** [reply]
查收 最好贴出来 方便别人看

**Comment by 746650396** [comment]
密钥怎么搞啊大佬，是这个吧："depots"
{
	"1868141"
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; {
		"DecryptionKey"		"dec1482ee88e2a62d7fddedf0e75c6abdef55d2884440de6508bef83044198d8"
	}
}


&nbsp;


……………………
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月24日(星期天) 下午5:15
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [pjy612/SteamManifestCache] 潜水员戴夫 (Discussion #149)






密钥呢？

—
直接回复此邮件，在 GitHub 上查看，或取消订阅。
***@***.***和&gt;
